### PR TITLE
Integration test to verify PubSub unsubscriptions

### PR DESCRIPTION
**Pull request by georgepsarakis**
## Description

Introduce an integration test to verify that PubSub channels Redis Backend are not increasing with time. See also #3812.

- [x] Add an integration task with a sleep time, in order to detect tasks in `STARTED` state.
- [x] Count the number of PubSub channels (subscriptions) before and after the chord execution.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -87,4 +87,4 @@ commands =
     pydocstyle {toxinidir}/celery
 
 [testenv:isort-check]
-commands = isort --project celery --order-by-type -rc -c {toxinidir}/celery {toxinidir}/t
+commands = isort --project celery --diff --order-by-type -rc -c {toxinidir}/celery {toxinidir}/t
```
Comment (thedrow) [inline tox.ini]: Why this change? If needed we can discuss it in a separate PR.

Comment (georgepsarakis) [reply tox.ini]: @thedrow it is not directly related, but it was quite difficult to understand why the `isort-check` build job was failing. By providing the diff, there is actual feedback to the developer about the changes that `isort` would perform in order to pass.

I agree that however small the change, optimally we should apply it separately. Let me know what works best in your opinion.

Comment (thedrow) [reply tox.ini]: Please pull this change to a different PR as it is not crucial for us to merge in order to release 4.2.

Comment (georgepsarakis) [reply tox.ini]: Opened #4469 for this purpose.

Comment (georgepsarakis) [reply tox.ini]: @thedrow I removed the commit and rebased.
